src/Rule/IndentationConfig.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace DTL\GherkinLint\Rule;
+
+use DTL\GherkinLint\Model\RuleConfig;
+
+class IndentationConfig implements RuleConfig
+{
+    public function __construct(
+        // number of spaces (or tabs) to use per indentation level
+        public int $width = 4,
+        public int $feature = 0,
+        public int $rule = 1,
+        public int $backgroud = 1,
+        public int $scenario = 1,
+        public int $step = 2,
+        public int $table = 3,
+        public int $literalBlock = 2,
+        public int $examples = 2,
+        public int $examplesTable = 3,
```

Comment:
based this on https://github.com/vsiakka/gherkin-lint/blob/master/src/rules/indentation.js

difference being that the values are levels, and the width is detmined by `$width` should work for tabs too.